### ISSUE TITLE
Application service API authentication uses query string for access token only?

### ISSUE THREAD

**Issue by jplatte** [matrix-org/matrix-doc#2734]
There's an existing issue about this at #1424, but it got closed by #1534 which updates the wording about the usage of the client-server API for appservices, but not the wording about authentication in the appservice API itself. This seems like an oversight.

**Comment by turt2live** [comment]
Which direction of traffic is your concern in? Homeserver -> Appservice or Appservice -> Homeserver?

**Comment by jplatte** [comment]
I think this is Homeserver > Appservice? [This](https://matrix.org/docs/spec/application_service/r0.1.2#authorization) is the section I'm talking about.

**Comment by turt2live** [comment]
Right, so https://github.com/matrix-org/matrix-doc/issues/1424 was about the other direction. It is true that the only way homeservers can prove their identity to an appservice is through the query parameters, and this would require an MSC to change.
